libtenzir/builtins/operators/every_cron.cpp

```diff
@@ -61,25 +75,24 @@ class every_operator final : public operator_base {
     if (auto* pipe = dynamic_cast<pipeline*>(result.replacement.get())) {
       auto ops = std::move(*pipe).unwrap();
       for (auto& op : ops) {
-        op = std::make_unique<every_operator>(std::move(result.replacement),
-                                              interval_);
+        op = std::make_unique<scheduled_execution_operator>(
+          std::move(result.replacement), std::move(scheduler_));
         // Only the first operator can be a source and needs to be replaced.
         break;
       }
       result.replacement = std::make_unique<pipeline>(std::move(ops));
       return result;
     }
-    result.replacement = std::make_unique<every_operator>(
-      std::move(result.replacement), interval_);
+    result.replacement = std::make_unique<scheduled_execution_operator>(
+      std::move(result.replacement), std::move(scheduler_));
```

Comment:
You are in a `const` member function, so you can't `std::move` the `scheduler_`.